### DISCUSSION TITLE
Please teach me coding animator

### DISCUSSION THREAD

**Discussion by Suppachok2527** [community/community#18853] [GitHub Education]
I want learning animete and make 3D polygon with coding and I wish learn machine learning i love blender 3D i love area maestosa i like it open RPG maker is nothing found in android

https://discord.gg/Nb2yY9PK8p

**Comment by Zerotask** [comment]
If you'd like to make 2D RPGs for Android etc. I can recommend you Bonfire for Flutter:
https://bonfire-engine.github.io/#/

**Reply by Suppachok2527** [reply]
We expect to be able to program and compiler using only android.But there are very few or almost nonexistent manuals.

**Reply by Suppachok2527** [reply]
We have no choice but to write a program.The laid-back gameplay must also allow for the sound font and sound bank to be played.just don't want the game to be too big than necessary Is there a tutorial on android solo gaming in the github community?I gave up expecting to use PC.If android can do all programming.I have ADHD but I have a lot of patience so it's better to make 3D models using programming. But now I'm lacking the basics.

**Comment by samus-aran** [comment]
Hey @Suppachok2527 

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're still looking for help for this specific topic, you might want to try asking for help from other [android game communities/repos](https://github.com/topics/android-game). It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck! ⭐
